op-devstack/sysgo/system.go

```diff
@@ -249,6 +249,64 @@ func DefaultInteropSystem(dest *DefaultInteropSystemIDs) stack.Option[*Orchestra
 	return opt
 }
 
+func DefaultIsthmusSuperProofsSystem(dest *DefaultInteropSystemIDs) stack.Option[*Orchestrator] {
```

Comment:
(nit) This looks basically identical to `DefaultInteropProofsSystem`, but without the call to `WithInteropAtGenesis()`.  Can we dedupe?

Reply:
indeed. done.